toolchain/check/handle_binding_pattern.cpp

```diff
@@ -28,14 +28,8 @@ auto HandleAddress(Context& context, Parse::AddressId parse_node) -> bool {
   return true;
 }
 
-auto HandleGenericBindingPattern(Context& context,
-                                 Parse::GenericBindingPatternId parse_node)
-    -> bool {
-  return context.TODO(parse_node, "GenericBindingPattern");
-}
-
-auto HandleBindingPattern(Context& context, Parse::BindingPatternId parse_node)
-    -> bool {
+auto HandleAnyBindingPattern(Context& context, Parse::NodeId parse_node,
+                             bool is_generic) -> bool {
```

Comment:
I think this should be:
```suggestion
                             bool is_comptime) -> bool {
```

Specifically, my understanding of the terminology is that the entity with the pattern becomes a generic thing, and these bindings result in generic parameters, but the bindings are "compile-time bindings".

(Somewhat commenting to see if I've internalized the terminology correctly, this isn't super important so feel free to defer some of this if needed. Also happy if we need to adjust the terminology.)

Reply:
I think you're right, but this is reflecting whether we had a `GenericBindingPattern` or a `BindingPattern` as the parse node, which is also using the old terminology. Probably best to update these names together.